packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import DotGovIcon from '../../images/usa-banner-dot-gov.svg';
+import HttpsIcon from '../../images/usa-banner-https.svg';
+import LockIcon from '../../images/usa-banner-lock.svg';
+import UsaFlagIcon from '../../images/usa-banner-flag.svg';
+
+const UsaBanner = function () {
+  const [isActive, setActive] = useState('false');
```

Comment:
useState doesn't works great but hcgov-design-system doesn't support it yet as it's on older version of react. Can revert to how header.jsx is currently keeping the state using class constructor.